tfrecords/create_tfrecords.py

```diff
@@ -11,7 +11,7 @@
 
 References:
 
-    * https://github.com/sayakpaul/TF-2.0-Hacks/blob/master/Cats_vs_Dogs_TFRecords.ipynb
+    * https://github.com/GoogleCloudPlatform/practical-ml-vision-book/blob/master/05_create_dataset/05_split_tfrecord.ipynb
     * https://www.tensorflow.org/tutorials/images/segmentation
 """
```

Comment:
Since this script can handle two different things (full-res and resized res), let's add a usage.

Reply:
sure thing!